vcell-rest/src/test/java/org/vcell/restq/apiclient/UsersApiTest.java

```diff
@@ -153,8 +155,12 @@ public void testOldAPITokenGeneration() throws ApiException {
         Assertions.assertEquals ("vcellNagios", token.getUserId());
 
         // Bob requests
-        token = bobUserApi.getLegacyApiToken();
-        assert (token.getToken() == null);
+        try{
+            bobUserApi.getLegacyApiToken();
+            Assertions.fail("Should throw 401 since only clients with role user can call it.");
```

Comment:
maybe change message that "should throw 401 since only users mapped to VCell Users can authenticate to this endpoint"

User bob does have role "user" it is just not mapped to a vcell user.
%test.quarkus.keycloak.devservices.users.bob=bob
%test.quarkus.keycloak.devservices.roles.bob=user